src/components/GPSButton/GPSButton.tsx

```diff
@@ -79,14 +81,17 @@ const GPSButton: FunctionComponent<GPSButtonProps> = ({
     <StyledButton
       className={className}
       data-testid="gpsButton"
-      icon={loading ? <LoadingIndicator color="black" /> : <GPSIcon />}
+      iconLeft={loading ? <LoadingIndicator color="black" /> : <GPSIcon />}
       aria-label="Huidige locatie"
       iconSize={20}
       onClick={onClick}
-      size={44}
+      size={144}
       variant="blank"
       type="button"
-    />
+    >
+      {' '}
```

Comment:
waarvoor dient deze spatie? kan het niet met een padding worden opgelost?

Reply:
Euh, dat is er per ongeluk ingekomen? Waarschijnlijk een left over van toen ik aan het kijken was of ik bij de juiste button aan het knutselen was.

Reply:
Ik heb het nu weggehaald.